src/modules/pages/components/Website/HomePage/Products/SlideDevelopers/SlideDevelopers.module.css

```diff
@@ -8,6 +8,6 @@
 
 .image {
   margin-bottom: 0;
-  max-height: 90%;
+  max-height: 98%;
```

Comment:
I think @collinvine's request for this to be larger was more along the lines of the code itself being larger rather than increasing the height. I think we need to take another screenshot. :grimacing: I told @reneegranillo that it would be ok to cut off the code wherever we need to as long as `createColony` was included.

@collinvine can you clarify your request?

Reply:
Maybe one of these might be better?

![Screenshot from 2019-08-28 12-48-43](https://user-images.githubusercontent.com/12519942/63849388-4580fc00-c992-11e9-8a73-6161ec8160cd.png)

![Screenshot from 2019-08-28 12-50-00](https://user-images.githubusercontent.com/12519942/63850225-e1f7ce00-c993-11e9-9bc5-97afe79d54c9.png)

Reply:
Ok. If it gets any bigger, we will cut off the `createColony` step so we will need to be more specific about what we want to show and whether we want it to show usable code.